src/blockfilter.h

```diff
@@ -45,19 +54,16 @@ class GCSFilter
 public:
 
     /** Constructs an empty filter. */
-    GCSFilter(uint64_t siphash_k0 = 0, uint64_t siphash_k1 = 0, uint8_t P = 0, uint32_t M = 0);
+    explicit GCSFilter(const Params& params = Params());
```

Comment:
Confused about why this is marked `explicit`...

Reply:
[Developer notes](https://github.com/bitcoin/bitcoin/blob/master/doc/developer-notes.md#c-data-structures): "By default, declare single-argument constructors `explicit`"